app.js

```diff
@@ -72,3 +72,8 @@ app.get("/busStops", (req, res, next) => {
     res.header("Access-Control-Allow-Origin", "*");
     res.json(busStops_data);
 });
+
+
+app.listen(PORT, () => {
+    console.log("Server running on port ",PORT);
```

Comment:
Can you remove the extra space here?
```suggestion
    console.log("Server running on port", PORT);
```